README.md

```diff
@@ -1,78 +1,406 @@
+The generation of representative test and simulation data is a challenging and time-consuming task. The DataHelix generator allows you to quickly create data, based on a JSON profile that defines fields and the relationships between them, for the purpose of testing and validation. The generator supports a number of generation modes, allowing the creation of data that both conforms to, or violates, the profile.
 
-You can use the generator to commission and maintain your test systems with data that conforms to your live environments without copying data or breaching GDPR.
+  - [Getting Started](#getting-started)
+    - [Creating your first profile](#creating-your-first-profile)
+    - [Adding constraints](#adding-constraints)
+    - [Generating large datasets](#generating-large-datasets)
+    - [Data types](#data-types)
+    - [Generation modes](#generation-modes)
+    - [Generating invalid data](#generating-invalid-data)
+    - [Next steps](#next-steps)
 
-The generator has been written in Java to allow cross platform compatibility, allowing it to work on Microsoft Windows, Apple Mac and Linux equally.
+# Getting Started
 
-[![CircleCI](https://circleci.com/gh/ScottLogic/datahelix.svg?style=svg)](https://circleci.com/gh/ScottLogic/datahelix)
+*The following guide gives a 10 minute introduction to the generator via various practical examples. For more detailed documentation please refer to the [Profile Developer Guide](???), and if you are interested in extending / modifying the generator itself, refer to the [DataHelix Generator Developer Guide](???).*
+
+The generator has been written in Java, allowing it to work on Microsoft Windows, Apple Mac and Linux. You will need Java v1.8 installed to run the generator (you can run `java version` to check whether you meet ths requirement), it can be [downloaded here](https://www.java.com/en/download/manual.jsp).
+
+The generator is distributed as a JAR file, with the latest release always available from the [GitHub releases page](https://github.com/ScottLogic/datahelix/releases/). The project is currently in beta and under active development. You can expect breaking changes in future releases, and new features too!
+
+Your feedback on the beta would be greatly appreciated. If you have any issues, feature requests, or ideas, please share them via the [GitHub issues page](https://github.com/ScottLogic/datahelix/issues).
+
+## Creating your first profile
+
+Profiles are JSON files that describe the data you want to generate. They are composed of:
+
+ - **fields** -  an array of uniquely named fields (or columns).
+ - **rules** - an array of constraints, with an optional description.
+ - **constraints** - reduce the data for a given column
```

Comment:
this description feels a little too concise. Ideas?

Reply:
```suggestion
 - **constraints** - restrict the types and ranges of data permitted for the given column
```